src/ln/channelmanager.rs

```diff
@@ -4462,24 +4441,82 @@ mod tests {
 		check_added_monitors!(nodes[1], 1);
 	}
 
+	#[test]
+	fn test_update_fee_that_funder_cannot_afford() {
+		let mut nodes = create_network(2);
+		let chan = create_announced_chan_between_nodes_with_value(&nodes, 0, 1, 1432, 200000);
+		let channel_id = chan.2;
+
+		let feerate = 260;
+		nodes[0].node.update_fee(channel_id, feerate).unwrap();
+		check_added_monitors!(nodes[0], 1);
+
+		let events_0 = nodes[0].node.get_and_clear_pending_msg_events();
+		assert_eq!(events_0.len(), 1);
+		let (update_msg, commitment_signed) = match events_0[0] {
+			MessageSendEvent::UpdateHTLCs {
+				node_id: _, updates: msgs::CommitmentUpdate {
```

Comment:
Note that you can generally replace "a_thing: _, b_thing" with "b_thing, .." to keep things shorter.

Reply:
noted